### DISCUSSION TITLE
Podcasts / Canal sobre desenvolvimento

### DISCUSSION THREAD

**Discussion by heliomsolivas** [frontendbr/forum#68] [Discussão]
Gostaria de saber se alguém tem alguma dica de Podcasts além das mais conhecidas (pelo menos pra mim) que é o [ZOFE](http://zofe.com.br/) e o [Grok Podcast](http://www.grokpodcast.com/), ou algum canal que fale sobre desenvolvimento como o [Update Sem Where](https://www.youtube.com/channel/UCZmp5CT7ASVdDkRZMzptLFw), acho que até em inglês é interessante se alguém tiver algum pra indicar.

**Comment by digiari** [comment]
Escutava bastante o ZOFE e o Grok, mas ultimamente estão parados... Tenho escutado o [DevNaEstrada](http://devnaestrada.com.br/), conteúdo top.
Também tem [esse repositório](https://github.com/ogilvieira/dev-podcast-list-brazil) com vários podcasts bacanas da área... serve? :)

**Comment by erichideki** [comment]
Essa lista pode te ajudar - https://github.com/erichideki/dev-podcast-list-brazil

**Comment by eduardojmatos** [comment]
@heliomsolivas indico o [DevNaEstrada](http://devnaestrada.com.br/) tbm do @femontanha e @raymonsanches

**Comment by felipefialho** [comment]
Sensacional sua lista @erichideki

**Comment by femontanha** [comment]
femontanha curtiu isso :+1:

**Comment by Cleysonlb** [comment]
https://www.youtube.com/user/LevelUpTuts  Um bom canal no youtube é o LevelUpTuts, é gringo e tem muito conteudo legal, alem do canal tem o site tbm!

**Comment by tcelestino** [comment]
@erichideki :+1:
